transformations/chinese_simplified_traditional_perturbation/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+import opencc
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+def chinese_simplified_traditional_perturbation(text,
+                          prob,
+                          seed,
+                          max_outputs,
+                          config
+                          ):
+    random.seed(seed)
+
+    perturbed_texts = []
+
+    if(config == 's2t'):
```

Comment:
Do you think that you could detect this automatically? This would be a big improvement to the PR I believe. 
I am not sure that all sentence from a corpus are either in simplified Chinese or in traditional Chinese so the automatic detection would improve the perturbation when working with Chinese content.

Reply:
Hmm, this would be interesting. I think the tricky bit will be how to categorize the words that are the same in both Simplified and Traditional Chinese. Sentences in Traditional Chinese can contain words that are considered Traditional but it will also be the same word in Simplified Chinese.

Reply:
maybe you could loop over the text and take the option which is in majority. 
E.g. if 70% of words are traditional chinese then we use t2s. Otherwise we use s2t.
What do you think about that?

Reply:
Hmm how would we count characters that are considered both Simplified and Traditional Chinese when looping over the text?  That would affect whether the text is majority Simplified or Traditional. 

Example for "Have you eaten yet?":

Simplified: 你吃饱了吗
Traditional: 你吃飽了嗎

As you can see, 3 out of the 5 characters are the same in both Simplified and Traditional which is "你", "吃", "了".

Therefore, I was thinking that it would still be better for the user to defined which transformation that they would like to do themselves. Now that I have also added the configurations that are available in the README file - they will know the options that are available. :)

What do you think @Nickeilf and @tongshuangwu ?